src/test/groovy/org/prebid/server/functional/testcontainers/Dependencies.groovy

```diff
@@ -24,7 +24,7 @@ class Dependencies {
             .withInitScript("org/prebid/server/functional/db_mysql_schema.sql")
             .withNetwork(network)
 
-    static final PostgreSQLContainer postgresqlContainer = new PostgreSQLContainer<>("postgres:16.0")
+    static final PostgreSQLContainer postgresqlContainer = new PostgreSQLContainer<>("postgres:12.18")
```

Comment:
What is this? Why did you try to fix the issue by rolling back to the oldest supported Postgres image? This doesn't look like a fix at all.

Reply:
Agreed; I added a dependency for PBS to resolve the error with basicJdbcClient. It was required for Vert.x. Now everything should work as intended